### DISCUSSION TITLE
Ayuda con el UI

### DISCUSSION THREAD

**Discussion by OscarIslasH** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#180] [Q&A]
Me la pase tratando de hacer muy estetico mi Wireframe y creo que lo que acabe haciendo fue el UI 

Igualmente, tengo la duda de saber como puedo hacer mi UI en base a mi Wireframe o si es mejor que tome mi Wireframe que ya hice como UI y hacer un Wireframe en base a eso.

A lo que entendi despues de ver videos donde hablan del tema una vez que acabe el wireframe es lo siguiente:

Wireframe: Es un boceto de como va a ir la pagina, la estructuracion de las cosas sin mucho diseño

UI (Interfaz de Usuario): Es como veria la pagina el usuario final una vez publicada la pagina, el diseño, la fuente de texto, explicaciones de que es lo que pasa cuando un boton se presiona, graficas, etc

Les paso mi "Wireframe" ya que creo que lo que acabe haciendo es el UI, si alguien pudiera aclararmelo, se los agradeceria: https://miro.com/app/board/uXjVOJwm_Ic=/?invite_link_id=430924955266

![Captura de pantalla 2022-03-01 144321](https://user-images.githubusercontent.com/99118114/156245770-dc534593-8d5a-416b-9090-30b342eb26b7.png)

**Comment by OmarVenturaP** [comment]
Esta muy bien tu Wireframe compañero, excelente! creo que si mataste 2 pájaros de un tiro. Cuanto tiempo te llevaste haciendolo?

**Reply by OscarIslasH** [reply]
¿Se pueden considerar ambos como el Wireframe y el UI? 

Gracias! me tarde como unas 20 horas en total, desde el 25 pude empezar a hacer el wireframe y por el trabajo fui haciendo de poco en poco, por que tampoco entendi muy bien y fui preguntando.

Me tarde mucho por que puse casi que 1000 objetos y pensando como ir poniendo cada cosa


![Captura de pantalla 2022-03-01 145602](https://user-images.githubusercontent.com/99118114/156247952-83630450-a064-4cd5-ba7c-eaf7c397044c.png)

**Reply by OmarVenturaP** [reply]
Que plan tienes, vas a seguir editando el UI? o lo dejarías así...
Porque si vas a seguir con el UI, entonces como recomendación (aunque soy un Novato también), apártalo en otro archivo, dejas este como el UX y el UI lo mejoras con colores, estilizados y demás, para que veas la diferencia del "BOCETO" y el resultado "FINAL". Que realmente es lo que separa a estos 2.

**Reply by OscarIslasH** [reply]
La verdad creo que es mejor usar lo que hice como UI y solo hacer un boceto como wireframe en balsamic, no dispongo de poco tiempo y aun acabando esto falta lo de HTML que hay que entregarlos hasta el 4

**Reply by OmarVenturaP** [reply]
Excelente, pues es una buena opción, ya para que el UX no te lleve demasiado tiempo pues menos detallado.
Pero vas muy bien compañero, síguele así.

Saludos

**Reply by OscarIslasH** [reply]
Ya mejor si estoy haciendo un UI en base al Wireframe jajaja : https://miro.com/app/board/uXjVOKNkGpc=/?invite_link_id=532521197083


![Captura de pantalla 2022-03-01 184328](https://user-images.githubusercontent.com/99118114/156273052-c3122ae8-ded9-4097-b958-9e0ad5b74224.png)

**Comment by JuanSalvador73** [comment]
Pues está bastante completo, hiciste 2 en 1. El UI es básicamente el mismo UX, solo que ya con el diseño gráfico "final", que te muestra como va a quedar la App finalmente. El UX representa la secuencia de pasos, la lógica del mismo sin entrar en detalles estéticos.
